api-report/fluid-static.api.md

```diff
@@ -86,6 +86,11 @@ export interface IMember {
     userId: string;
 }
 
+// @public
+export type IMyself<M extends IMember = IMember> = M & {
```

Comment:
Can you make this type `Myself` instead of IMyself.

```suggestion
export type Myself<M extends IMember = IMember> = M & {
```

Reply:
I know the member type is IMember but it's designed to be augmented by the inherting class. So the actual return type is `AzureMember` not IMember. for the Myself case `Myself<AzureMember>` is better than `IMyself<AzureMember>`

In addition we usually use `I` to represent interfaces that are designed to be implemented by a class not defined as a type.